apps/examples/stack_protection/stack_protection_main.c

```diff
@@ -0,0 +1,76 @@
+/****************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ * Author: Sangamanatha <sangam.swami@samsung.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <tinyara/config.h>
+
+#include <sys/types.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <errno.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+#ifdef CONFIG_BUILD_KERNEL
+int main(int argc, FAR char *argv[])
+#else
+int stack_prot_tc_main(int argc, char *argv[])
+#endif
+{
+	char ch;
+	int i = 0;
+	volatile char array_normal[10];
+	volatile char array_overflow[CONFIG_TASH_CMDTASK_STACKSIZE];
```

Comment:
1. You set 512 for stacksize in makefile so that `CONFIG_TASH_CMDTASK_STACKSIZE` is not used. And this should be working with TASH disabled.
Please use another value.
2. Let's use new definitions for those values. 10 -> xxx, config_tash_xx -> xxxxx

Reply:
1. Changed the stack size to 1024.
2. yes changed to macros